Add 'testers_groups' option to 'testfairy' action

farshadtx opened the pull request:
This allows the ability to specify the testing groups that will be notified when uploading a build to TestFairy.

farshadtx commented:
See the added issue: #4212

jgreubel commented:
Any update on this?  This would be a nice quality of life change!

mfurtak commented:
Hi @farshadtx - thanks for the contribution 👍 

It would be great to have a new test added to `testfairy_spec.rb` that asserts this behavior. Let me know if you need help with that!

mfurtak commented on fastlane/lib/fastlane/actions/testfairy.rb:
```diff
@@ -67,7 +67,11 @@ def self.available_options
+          FastlaneCore::ConfigItem.new(key: :testers_groups,
+                                       env_name: "FL_TESTFAIRY_TESTERS_GROUPS",
+                                       description: "Comma seperated list of tester groups to be notified",
+                                       default_value: "") # the default value is an empty list
```
It would be good to represent this as an Array type option if it can take multiple values.

```ruby
type: Array,
default_value: []
```

Then, you can use `join(',')` to pass the values as a single string

jgreubel commented:
Hey, we are working on our own projects at work.  We will try to attend to this soon.  Please do not close this issue/pull request! Thanks!

jgreubel commented:
Just updated!  Let me know what you think!

mfurtak commented:
Looks good! Please squash down to 1 commit and I'll get it in 👍

Commit:
[testfairy] Add 'testers_groups' option to 'testfairy' action

- Change testers_group type to array
- Add optional parameters test
- Add short option value

Issue: #4212

Signed-off-by: Farshad Sheykhi <fsheykhi@tillster.com@>

jgreubel commented:
I think I've done that.  Let me know!

mfurtak commented:
It seems like Danger is still having trouble with squashes. 😞 

One more try: can you try rebasing your branch against master and then force pushing this branch?

Thanks!

farshadtx commented:
Please check our new [PR](https://github.com/fastlane/fastlane/pull/4425)
